op-e2e/setup.go

```diff
@@ -624,32 +628,48 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		return nil, err
 	}
 
-	for name := range cfg.Nodes {
-		if name == RoleL1 {
-			return nil, fmt.Errorf("node name %s is reserved for L1 node", RoleL1)
-		}
+	// Ordered such that the Sequencer is initialized first. Setup this way so that
+	// the `RollupSequencerHTTP` GethOption can be supplied to any sentry nodes.
+	l2Nodes := []string{RoleSeq, RoleVerif}
+	for _, name := range l2Nodes {
 		var ethClient services.EthInstance
 		if cfg.ExternalL2Shim == "" {
-			l2Geth, err := geth.InitL2(name, l2Genesis, cfg.JWTFilePath, cfg.GethOptions[name]...)
+			opts := cfg.GethOptions[name]
+			if name != RoleSeq && !cfg.DisableTxForwarder {
+				opts = append(opts, func(ethCfg *ethconfig.Config, nodeCfg *node.Config) error {
+					ethCfg.RollupSequencerHTTP = sys.EthInstances[RoleSeq].UserRPC().RPC()
+					cfg.Loggers[name].Warn("URL OPTION SET", "url", ethCfg.RollupSequencerHTTP)
+					return nil
+				})
+			}
+
```

Comment:
This is broken. For the new option to be picked up by the system, you need to add `cfg.GethOptions[name] = opt` to it. Since the slice header length changed, and that isn't written to the slice header value within the map.

Reply:
Edit: map is not used after, so not a bug, but does get out of sync with the opts map